src/App.tsx

```diff
@@ -1,13 +1,13 @@
-import EventLoop from './components/EventLoop/EventLoop.tsx';
-import Editor from './components/Editor/Editor.tsx';
-import TasksQueue from './components/TasksQueue/TasksQueue.tsx';
-import Console from './components/Console/Console.tsx';
+import * as Styled from './App.styled.ts';
```

Comment:
Please don't mind the changes you see in the import section.

My Vscode cleans up the order of imports on saving the changes to a file.
I did not make any new imports here.